guide/C/ch_currency.xml

```diff
@@ -437,56 +443,44 @@ TODO -->
           exchange rates for that currency anymore, do the following:
         </para>
 
-        <itemizedlist>
-          <listitem>
-            <para>Open the Securities window by selecting
-              <menuchoice>
-                <guimenu>Tools</guimenu><guimenuitem>Security Editor</guimenuitem>
-              </menuchoice>
-              .
-            </para>
-          </listitem>
+        <procedure>
+          <step><simpara>Open the Securities window by selecting <menuchoice>
+                <guimenu>Tools</guimenu><guimenuitem>Security Editor</guimenuitem></menuchoice>.</simpara>
+          </step>
 
-          <listitem>
-            <para>Make sure the <guilabel>Show National Currencies</guilabel> box is selected.
-            </para>
-          </listitem>
+          <step><simpara>Make sure the <guilabel>Show National Currencies</guilabel> box is selected.</simpara></step>
 
-          <listitem>
-            <para>Expand the CURRENCY row.
-            </para>
-          </listitem>
+          <step><simpara>Expand the <guilabel>CURRENCY</guilabel> row.</simpara></step>
 
-          <listitem>
-            <para>Double click on the currency for which you want to disable exchange rate retrieval.
-            </para>
-          </listitem>
+          <step><simpara>Double click on the currency for which you want to disable exchange rate retrieval.</simpara></step>
```

Comment:
I was long searching. Either we should 

1. display the column in the program by default or
2. Remind the reader to display the "Get Quotes" column.

@jralls et al.: your opinion?